source/documentation/services/authentication.html.md.erb

```diff
@@ -37,6 +39,31 @@ Any changes to that array can be tested directly in the interface. Be aware that
 
 Below the above rule you'll also find the `Slack Notification on User Signup` rule which sends a message to the `#form-builder-deployments` channel whenever a new user signs up. It requires the `SLACK_HOOK_URL` to be set in order for it to work.
 
+### How to add a domain to the list
+
+In order to add a domain to the list, we have to add the corresponding string to the constant
+
+```
+ALLOWED_DOMAINS
+```
+
+in the configuration file: [config/initializers/allowed_domains.rb](https://github.com/ministryofjustice/fb-editor/blob/main/config/initializers/allowed_domains.rb)
+
+
+Currently it has the 5 domains :
```

Comment:
Are we planning to maintain this list as we go? If not I'd probably remove it as it will just drift out of sync over time and we already link to the source file